modules/rostests/rosautotest/main.cpp

```diff
@@ -87,6 +89,15 @@ extern "C" int
 wmain(int argc, wchar_t* argv[])
 {
     int ReturnValue = 1;
+    DWORD TestStartTime, TestEndTime;
```

Comment:
More explicit, like
```suggestion
    DWORD TestRunStartTime, TestRunEndTime;
```

Reply:
It's already clear what "Start" and "End" means in this context.